### PR TITLE
Bug/dtserwfour 373 remove empty state button

### PR DESCRIPTION
**Pull request by pdamodarannair**

### CODE REVIEW

File: transfermethodui/src/main/res/layout/fragment_list_transfer_method.xml [hunk 1]
```diff
@@ -44,19 +44,6 @@
...
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_horizontal|center"
-                android:layout_marginStart="@dimen/grid_margin_left"
-                android:layout_marginTop="@dimen/grid_margin_top"
-                android:layout_marginEnd="@dimen/grid_margin_right"
-                android:layout_marginBottom="@dimen/grid_margin_bottom"
-                android:maxWidth="@dimen/create_button_max_width"
-                android:text="@string/mobileAddTransferMethodHeader" />
         </LinearLayout>
```
Comment (malaw-hw) [inline transfermethodui/src/main/res/layout/fragment_list_transfer_method.xml:47]: please update the gradle file to use a right version.
  project.version = "1.0.0-beta10-SNAPSHOT"
  hyperwalletCoreVersion = '1.0.0-beta09-SNAPSHOT'

**Review by malaw-hw — CHANGES_REQUESTED** [review]

File: transfermethodui/src/main/res/layout/fragment_list_transfer_method.xml (hunk 1 above)
Comment (pdamodarannair) [reply transfermethodui/src/main/res/layout/fragment_list_transfer_method.xml:47]: resolved

**Review by shanavas123 — APPROVED** [review]

File: transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodFragment.java [hunk 2]
```diff
@@ -162,13 +162,6 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         super.onViewCreated(view, savedInstanceState);
         mProgressBar = view.findViewById(R.id.list_transfer_method_progress_bar_layout);
         mEmptyListView = view.findViewById(R.id.empty_transfer_method_list_layout);
-        Button button = view.findViewById(R.id.add_account_button);
-        button.setOnClickListener(new OneClickListener() {
-            @Override
-            public void onOneClick(View v) {
-                mOnAddNewTransferMethodSelected.showSelectTransferMethodView();
-            }
-        });
```
Comment (vwagh-hw) [inline transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodFragment.java:171]: Are the UI tests passing after removing this?

Comment (pdamodarannair) [reply transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodFragment.java:171]: UI tests are passing after this. Please find the screenshot. 
Screenshot 2021-02-26 at 4 05 06 PM

**Review by shanavas123 — APPROVED** [review]

**Review by SaiNarendran — APPROVED** [review]

**Review by malaw-hw — APPROVED** [review]

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**